UNRELEASED.md

```diff
@@ -1,3 +1,6 @@
 # Unreleased
 
-_Nothing yet!___
+**Added:**
+
+- react-native-bpk-component-select:
+  - Added validation options consistent with those for text inputs.
```

Comment:
Can we make this a bit more explicit? I like the note about consistency but it'd be more helpful to users if we specifically stated which props are new. No harm in popping a link to the page on the docs site too.